src/pyflex/window_selector.py

```diff
@@ -232,12 +247,12 @@ def _parse_event_and_station(self):
         # Last resort, if either is not set, and the observed or synthetics
         # are sac files, get the information from there.
         if not self.station or not self.event:
-            if hasattr(self.observed.stats, "sac"):
-                tr = self.observed
-                ftype = "observed"
-            elif hasattr(self.synthetic.stats, "sac"):
+            if hasattr(self.synthetic.stats, "sac"):
```

Comment:
I think this is a personal choice then...

For my case, the observed data(sac format) usually doesn't contain the event information. So When you try to extract event information from the observed file, it will always fail. On the other hand, the synthetic is generated by specfem so the header information is more complete.

So what do you think?